techniques/html/H34.html

```diff
@@ -98,38 +97,34 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
       
       
       <section id="description">
-         <h2>Description</h2>
-         <p>The objective of this technique is to use Unicode right-to-left marks and left-to-right
-            marks to override the HTML bidirectional algorithm when it produces undesirable results.
-            This may be necessary, for instance, when placing neutral characters such as spaces or
-            punctuation between different directional text runs. The concepts used in this technique
-            are described in <a href="https://www.w3.org/International/articles/inline-bidi-markup/">Inline markup and bidirectional text in HTML</a>.</p>
-            <p>Unicode right-to-left marks and left-to-right marks can be entered directly or by means of character entities or numeric character references, as shown here.</p>
+         <h2>解説</h2>
+         <p>このテクニックの目的は、HTML の双方向性アルゴリズムが望ましくない結果を生じるときに、それを無効にするために Unicode 制御文字の right-to-left mark と left-to-right mark を用いることである。たとえば、スペース又は句読点のようなニュートラルな文字が異なる方向性のテキストの間に置かれている時に必要となることがある。この達成方法で用いられているコンセプトは、W3C 文書の <a href="https://www.w3.org/International/articles/inline-bidi-markup/">Inline markup and bidirectional text in HTML</a> に書かれている。</p>
+            <p>Unicode 制御文字の right-to-left mark 及び left-to-right mark は、以下に示すように、そのまま、又は文字符号か数字符号の参照によって挿入することが可能である。</p>
       <ul>
          <li>left-to-right mark: <code class="language-html">&amp;lrm;</code> or <code>&amp;#x200e;</code> (<code>U+200E</code>)</li>
          <li>right-to-left mark: <code class="language-html">&amp;rlm;</code> or <code>&amp;#x200f;</code> (<code>U+200F</code>)</li>
```

Comment:
この or は、訳したほうがよさそうでしょうか。
```suggestion
         <li>right-to-left mark: <code class="language-html">&amp;rlm;</code> 又は <code>&amp;#x200f;</code> (<code>U+200F</code>)</li>
```